contracts/test/plugin/proposal/proposal.ts

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
There were no tests on aragon/osx. I created a task for this: https://aragonassociation.atlassian.net/browse/OS-791